CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 ## Unreleased
 
+## 1.7.1
+
 - Fix: Report correct remaining tasks until next area unlock in Leagues notifications. (#369)
```

Comment:
```suggestion
- Bugfix: Report correct remaining tasks until next area unlock in Leagues notifications. (#369)
```

fix is more correct but i don't wanna start now with where to sort fix vs bugfix